### ISSUE TITLE
Support for mounting a PVC for a transformer

### ISSUE THREAD

**Issue by jkwak3** [kubeflow/kfserving#720]
/kind feature
I saved a sklearn scaler object as python pickle in a pvc before training. After training, I wanted to load the object in kf serving transformer pod so that a proper preprocessing can be done. However, I could not find any examples of mounting a pvc for transformer. If there is a way to mount a pvc, I would like to know how. If not, I think it is necessary to support the feature.

**Describe the solution you'd like**
[A clear and concise description of what you want to happen.]

There should be a way to mount a pvc for a transformer pod.

**Anything else you would like to add:**
[Miscellaneous information that will assist in solving the issue.]

**Comment by yuzisun** [comment]
please check https://github.com/kubeflow/kfserving/issues/664, you should be able to mount the pvc by adding env `STORAGE_URI` if you pull from master.

**Comment by yuzisun** [comment]
@jkwak3 this feature is released under 0.3 now, please give a try by adding following env on your transformer spec.
```
env:
  - name: STORAGE_URI 
     value: pvc://kfserving-pvc/models
```

**Comment by yuzisun** [comment]
Please reopen if this does not work for you.
/close
